Building with feature DIPOLES but without ELECTROSTATICS fails

As suggested by the issue title, building fails when compiling with feature DIPOLES &ndash; unless the ELECTROSTATICS feature is also compiled in. Produces error message
```
In file included from /tikhome/pkreissl/src/espresso/src/core/electrostatics_magnetostatics/dipole_inline.hpp:23:0,
                 from /tikhome/pkreissl/src/espresso/src/core/energy_inline.hpp:69,
                 from /tikhome/pkreissl/src/espresso/src/core/energy.cpp:30:
/tikhome/pkreissl/src/espresso/src/core/electrostatics_magnetostatics/p3m-dipolar.hpp:95:3: error: ‘p3m_send_mesh’ does not name a type; did you mean ‘p3m_local_mesh’?
   p3m_send_mesh sm;
   ^~~~~~~~~~~~~
   p3m_local_mesh
```
.

--- a/src/core/electrostatics_magnetostatics/p3m_send_mesh.cpp
+++ b/src/core/electrostatics_magnetostatics/p3m_send_mesh.cpp
@@ -20,7 +20,8 @@
  */
 #include "p3m_send_mesh.hpp"
 
-#ifdef P3M
+#if defined(P3M) || defined(DP3M)
+
 #include "fft.hpp"
 
 #include <utils/mpi/cart_comm.hpp>

--- a/src/core/electrostatics_magnetostatics/p3m_send_mesh.hpp
+++ b/src/core/electrostatics_magnetostatics/p3m_send_mesh.hpp
@@ -23,7 +23,7 @@
 
 #include "p3m-common.hpp"
 
-#ifdef P3M
+#if defined(P3M) || defined(DP3M)
 
 #include <utils/Span.hpp>
 
@@ -78,6 +78,5 @@ public:
     spread_grid(Utils::make_span(&mesh, 1), comm, dim);
   }
 };
-
 #endif
 #endif // ESPRESSO_P3M_SEND_MESH_HPP

--- a/testsuite/python/dawaanr-and-bh-gpu.py
+++ b/testsuite/python/dawaanr-and-bh-gpu.py
@@ -30,7 +30,7 @@ def stopAll(system):
 
 
 @utx.skipIfMissingGPU()
-@utx.skipIfMissingFeatures(["DIPOLAR_BARNES_HUT"])
+@utx.skipIfMissingFeatures(["DIPOLAR_BARNES_HUT", "LENNARD_JONES"])
 class BHGPUTest(ut.TestCase):
     system = espressomd.System(box_l=[1, 1, 1])
     system.seed = system.cell_system.get_state()['n_nodes'] * [1234]

--- a/testsuite/python/dawaanr-and-dds-gpu.py
+++ b/testsuite/python/dawaanr-and-dds-gpu.py
@@ -26,7 +26,7 @@ import espressomd.analyze
 
 
 @utx.skipIfMissingGPU()
-@utx.skipIfMissingFeatures(["DIPOLES", "ROTATION"])
+@utx.skipIfMissingFeatures(["DIPOLES", "ROTATION", "LENNARD_JONES"])
 class DDSGPUTest(ut.TestCase):
     # Handle for espresso system
     es = espressomd.System(box_l=[1.0, 1.0, 1.0])

--- a/testsuite/python/dds-and-bh-gpu.py
+++ b/testsuite/python/dds-and-bh-gpu.py
@@ -31,7 +31,7 @@ def stopAll(system):
 
 
 @utx.skipIfMissingGPU()
-@utx.skipIfMissingFeatures(["DIPOLAR_BARNES_HUT"])
+@utx.skipIfMissingFeatures(["DIPOLAR_BARNES_HUT", "LENNARD_JONES"])
 class BH_DDS_gpu_multCPU_test(ut.TestCase):
     system = espressomd.System(box_l=[1, 1, 1])
     # just some seeding based on 14

